graylog2-server/src/main/java/org/graylog2/inputs/codecs/SyslogCodec.java

```diff
@@ -91,10 +89,7 @@ public Message decode(@Nonnull RawMessage rawMessage) {
                 remoteAddress = address.getInetSocketAddress();
             }
             return parse(msg, remoteAddress == null ? null: remoteAddress.getAddress(), rawMessage.getTimestamp());
-        } catch (ClassCastException e) {
```

Comment:
It didn't do anything. [`ClassCastException`](https://docs.oracle.com/javase/8/docs/api/java/lang/ClassCastException.html) is a `RuntimeException` and the `catch` clause simply propagated (i. e. re-threw) the exception.